apps/tlon-mobile/src/screens/ActivityScreen.tsx

```diff
@@ -50,15 +56,30 @@ export function ActivityScreen(props: Props) {
   return (
     <ContactsProvider contacts={contacts ?? []}>
       <View backgroundColor="$background" flex={1}>
-        <ActivityScreenView
-          bucketFetchers={bucketedActivity}
-          isFocused={isFocused}
-          goToChannel={handleGoToChannel}
-          goToThread={handleGoToThread}
-          refresh={handleRefreshActivity}
-        />
+        <ErrorBoundary fallback={<ErrorFallback />}>
```

Comment:
Could we add this at the root of the app instead?

Reply:
Let's do both for now. I can update this to use the same component you have once that PR merges